src/index.tsx

```diff
@@ -659,6 +685,7 @@ class App extends React.Component<{}, AppState> {
                 // if no element is clicked, clear the selection and redraw
                 if (draggingElement === null) {
                   clearSelection();
+                  pushHistoryCurrentEntry();
```

Comment:
If I remember, this if is hit only when there is a click within an empty area, is correct to push the history in this case? We are calling `JSON.stringify` two times on each call of `pushHistoryCurrentEntry`.
It's only an optimization, not required now.

Reply:
I wrote NotForCommit because I’m unsure about this part. I’m not exactly sure when we should be pushing the history entry